src/cdk/drag-drop/drag-ref.ts

```diff
@@ -1044,10 +1044,27 @@ export class DragRef<T = any> {
         // we can get away with it. See https://bugzilla.mozilla.org/show_bug.cgi?id=1615824.
         (event.touches[0] || event.changedTouches[0] || {pageX: 0, pageY: 0}) : event;
 
-    return {
-      x: point.pageX - scrollPosition.left,
-      y: point.pageY - scrollPosition.top
+    let point = {
+      x: eventPoint.pageX - scrollPosition.left,
+      y: eventPoint.pageY - scrollPosition.top
     };
+
```

Comment:
We could avoid creating the object unnecessarily for SVGs by doing something like this:

```
const x = eventPoint.pageX - scrollPosition.left;
const y = eventPoint.pageY - scrollPosition.top;

if (isSvg) {
  // SVG-specific logic.
  return svgPoint;
}

return {x, y};
```